pkg/cli/cmds/k3sopts.go

```diff
@@ -19,10 +19,11 @@ var (
 var copy *K3SFlagOption = nil
 
 type K3SFlagOption struct {
-	Hide    bool
-	Drop    bool
-	Usage   string
-	Default string
+	Hide            bool
+	Drop            bool
+	Usage           string
+	Default         string
+	OverrideDefault bool
```

Comment:
Needed to allow setting the default to an empty string. Could also just remove the empty string check, but I assumed it was there for a reason?